### PR TITLE
Define Automatic-Module-Name for Retrofit modules

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: retrofit-adapters/scala/pom.xml [hunk 1]
```diff
@@ -69,6 +69,17 @@
           </signature>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>retrofit2.converter.scala</Automatic-Module-Name>
```
Comment (swankjesse) [inline retrofit-adapters/scala/pom.xml]: I’m continuing a problem: the Scala adapter uses this package: `retrofit2.converter.scala`.

Comment (swankjesse) [reply retrofit-adapters/scala/pom.xml]: Worth the grind to move it to the right package in follow up?

Comment (JakeWharton) [reply retrofit-adapters/scala/pom.xml]: It's unreleased, so yes.

Comment (swankjesse) [reply retrofit-adapters/scala/pom.xml]: https://github.com/square/retrofit/pull/2639

**Pull request merged**